docs/api/app.md

```diff
@@ -218,6 +218,30 @@ bar, and on Mac you can visit it from dock menu.
 
 Clears the recent documents list.
 
+## app.clearCache(begin, end. callback)
```

Comment:
`app.clearCache(begin, end. callback)` should be `app.clearCache(begin, end, callback)`